packages/helpers/src/error/index.ts

```diff
@@ -0,0 +1,22 @@
+import { ShopwareError } from "@shopware-pwa/commons/interfaces/errors/ApiError";
+
+/**
+ * Get the messages from the API response (array of ShopwareErrors)
+ */
+export function getMessagesFromErrorsArray(
+  errors: ShopwareError[]
+): String[] | undefined {
+  if (!errors || !Array.isArray(errors) || !errors.length) {
+    return;
+  }
+  // return a message only if detail and pointer propery is provided in upcoming ShopwareError
+  return errors
+    .map(
+      ({ detail, source }) =>
+        detail &&
+        source &&
+        source.pointer &&
+        `${source.pointer.substring(1)}: ${detail}`
+    )
+    .filter(message => message);
```

Comment:
this is not doing anything :)
```suggestion
```

Reply:
the output of .map function is possible of type "undefined" (due to condition) so it's just filtering out all the falsy values :) but now I can see the better solution would be .filter(String) to get only the strings...